Assets/Scripts/Interactables/ItemPickup.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Managers;
+
+public class ItemPickup : Interactable
+{
+    [SerializeField] private Item item; //reference for items
+
+    /// <summary>
+    /// New function that override the Interact function from Interactable Class
+    /// </summary>
+    public override void Interact()
+    {
+        PickUp();
+    }
+
+    /// <summary>
+    /// Function for picking up item, add to inventory and then make item disappeared
+    /// </summary>
+    public void PickUp()
+    {
+        //Add to inventory
+        PlayerManager.Instance.GetInventory().Add(item); 
```

Comment:
Can an item be picked up multiple times, for example after throwing it away from inventory? Then I think you should get the inventory once, and then always add to it. Don't get it via the playermanager > player every time the item is picked up.

Reply:
Yeah changed to this 
```suggestion
        private void Awake()
    {
        GetInventory();
    }

    private Inventory GetInventory()
    {
        return inventory = PlayerManager.Instance.GetPlayer().GetComponentInChildren<Inventory>();
    }
```
should be working fine

Reply:
For now i guess we are not able to drop the item yet so i can't test it but im pretty sure it will work just fine.

Reply:
thanks!